runtime/j9vm/jvm.c

```diff
@@ -1657,6 +1658,7 @@ addHarmonyPortLibrary(J9PortLibrary * portLib, J9JavaVMArgInfoList *vmArgumentsL
 	addHarmonyPortLibToVMArgs(portLib, &(optArg->vmOpt), &dummyArgs, &harmonyPortLibrary);
 	return 0;
 }
+#endif
```

Comment:
Please add the `/* J9VM_OPT_HARMONY */` comment to all the #endif's in this changeset.

Reply:
Good idea.  Done on the files in this changeset only.